src/templates/DotnetTemplateRetriever.ts

```diff
@@ -80,7 +81,7 @@ export function getDotnetVerifiedTemplateIds(runtime: string): string[] {
             case ProjectRuntime.beta:
                 return `${id}.2.x`;
             default:
-                throw new RangeError();
+                throw new RangeError(localize('invalidRuntime', 'Invalid runtime "{0}".', runtime));
```

Comment:
Does localize handle it if runtime is undefined?  Maybe consider String(runtime)

Reply:
Yeah it's handled